### DISCUSSION TITLE
Which all standard Aws services is required  to deploy server pod

### DISCUSSION THREAD

**Discussion by melwyn-castelino** [serverpod/serverpod#570] [Q&A] [answered]
@vlidholt @yahu1031 

Hi as I am new to server pod 
Can you please elaborate the list of services which would be required  by the prod app  ?

I have added these many services as per my understanding . 
https://calculator.aws/#/estimate?nc2=h_ql_pr_calc&id=5c0471cce5891321618ca61403f63980d158599c

**Comment by vlidholt** [comment, marked as answer]
That looks about correct. If you are concerned about costs, ElastiCache is optional, and it's also possible to choose a less expensive database setup. If you are eligible for the free tier, a Serverpod deployment should fit within it.
